index.html

```diff
@@ -0,0 +1,199 @@
+              </li>
+            </ul>
+            <ul class="program-order program-order-last">
+              <li class="program-list program-icon">
+                <img class="program-images" src="./assets/images/workshop.png">
+              </li>
+              <li class="program-list program-name">Oda Workshop</li>
+              <li class="program-list progrm-description txt-main">
+                <p>
+                  Create your own small business in preparing oromoo cultural food  for participants and promoting unique  cultural food beyond celebrating the festival 
+                </p>
+              </li>
+            </ul>
+            <ul class="program-order">
+              <li class="program-list program-icon">
+                <img class="program-images" src="./assets/images/ignite.png">
+              </li>
+              <li class="program-list program-name">Erriecha Ignite </li>
+              <li class="program-list progrm-description txt-main">
+                <p>
+                  Get opportunities  to connect with “Follee” Oromoo youth around the world to promote Irreechaa festival  afterthis celebration.
+                </p>
+              </li>
+            </ul>
+           </div>
+           <h2 class="program-join mobile-only">Join CC Global Submit 2024</h2>
+           <h2 class="program-details desktop-only">See The Whole Program</h2>
+        </div>
+      </div>
+      </section>
+      <section class="section-speaker">
+         <h1 class="head-speaker start-align end-align">Featured Speaker</h1>
+         <div class="speaker-container start-align end-align" id="featurespeakerid">
+          
+         </div>
+      </section>
+    </main>
+    <footer class="">
+      <section class="footer-partner bg-main">
+        <div class="partner-head start-align end-align">
+          <h2 class="partner-text txt-second">
+            Partner
+           </h2>
+        </div>
+         <div class="partner-log start-align end-align txt-main">
+             <img class="partner-image" src="./assets/images/partner/abyssiniabank.png">
+             <img class="partner-image" src="./assets/images/partner/awashbank.svg">
+             <img class="partner-image" src="./assets/images/partner/nibbank.png">
+             <img class="partner-image" src="./assets/images/partner/oromiabank.png">
+             <img class="partner-image" src="./assets/images/partner/microverse.png">
+         </div>
+      </section>
+      <section class="footer-logo bg-second ">
+         <div class="footerlogo-icon start-align">
+            <img src="./assets/images/titleicon.png" alt="Festival logo" class="event-logo">
+          </div>
+          <div class="footerevent-message end-align txt-main">
+             <p>Irreechaa  Love,Peace and Unity</p>
+             <p>
+              <br>
+              CELEBRATING IRREECHAA FESTIVAL 2023
+             </p>
+          </div>
+         </div>
+      </section>
+    </footer>
```

Comment:
You're required to stick to the guiding design as close as possible. This includes color, background-color, layout, padding, margin among others. You must persionalize the content, but the layout, color, layout must remain. Kindly walk your design alongside the guiding design


### Your Design
![your-partner-footer-home](https://user-images.githubusercontent.com/26597432/226095991-14c4bb5e-4269-45d0-9efc-c23e4d6e66fa.png)

### Template Design
![temp-partner-footer-home](https://user-images.githubusercontent.com/26597432/226096002-02640519-df10-4e3a-96fa-c93f0cbbeeea.png)